docs/azure_jumpstart_ag/contoso_supermarket/arc_defender_servers/_index.md

```diff
@@ -0,0 +1,94 @@
+
+## Enable Microsoft Defender for Servers
+
+To protect workloads with Microsoft Defender for Servers, you will need to enable its Servers plan at subscription level. Follow these steps, if it is not already enabled:
+
+- Login to AZ CLI using the ```az login``` command.
+
+- Ensure that you have selected the correct subscription by using the ```az account list --query "[?isDefault]"``` command. If you need to adjust the active subscription used by Az CLI, follow [this guidance](https://docs.microsoft.com/cli/azure/manage-azure-subscriptions-azure-cli#change-the-active-subscription).
+
+- Run the following command to check whether Microsoft Defender for Servers is already enabled in your subscription or not:
+
+    ```shell
+    az security pricing show -n VirtualMachines
+    ```
+
+    ![Screenshot of checking Microsoft Defender for Servers enabled at subscription level](./img/01.png)
+
+    > **NOTE: Proceed with the next step if the value you have got for _pricingTier_ is equals to _Free_. If the value was _Standard_, Microsoft Defender for Servers is already enabled and no additional action is required.**
+
+- Run the following command to enable Microsoft Defender for Servers in your subscription:
+
+    ```shell
+    az security pricing create -n VirtualMachines --tier 'standard'
+    ```
+
+    ![Screenshot of enabling Microsoft Defender for Servers at subscription level](./img/02.png)
+
+    > **NOTE: Remember to follow the steps provided in the Cleanup section to disable Microsoft Defender for Servers, otherwise you may incur additional charges in your subscription.**
+
+## Azure Arc-enabled servers and Microsoft Defender for Servers Integration
+
+Now that you have successfully enabled Microsoft Defender for Servers, you will get recommendations to help you protect your resources, including your Azure Arc-enabled servers.
+
+Microsoft Defender for Servers will collect data from your Arc-enabled servers to monitor for security vulnerabilities and threats. The data collection will allow greater visibility into missing updates, non-secure OS settings, endpoint protection status, health and threat protection.
+
+- Navigate to Microsoft Defender for Cloud. In the **Inventory** section under **VM and Servers**, Defender will provide you with an overview of all the discovered security recommendations for your **Azure Arc-enabled servers**.
+
+    ![Screenshot of Microsoft Defender for Cloud Inventory](./img/03.png)
+
+    > **NOTE: it may take up to 30 minutes for your Azure Arc-enabled server to be shown in Microsoft Defender for Cloud Dashboard**
```

Comment:
Any troubleshooting steps if it doesn't?

If so, add a Troubleshooting section as discussed